spec/features/merchants/invoices/show_spec.rb

```diff
@@ -0,0 +1,20 @@
+require "rails_helper"
+
+RSpec.describe "Merchants Invoice Show Page" do 
+  #user story 15
+  before(:each) do 
+    top_customer_data
+  end
+
+  it "has information related to the invoice" do 
+    visit "/merchants/#{@merchant.id}/invoices/#{@invoice_3.id}"
+    # visit merchant_invoices_path(@merchant, @invoice_3)
+    within "#invoice_info" do 
+
+    expect(page).to have_content("Invoice ID: #{@invoice_3.id}")
```

Comment:
might be useful to have a a couple `to_not` expect statements to ensure the show page doesn't have data from another invoice